oomcli/cmd/get_meta_feature.go

```diff
@@ -64,11 +68,33 @@ func printFeatures(features types.FeatureList, output string, wide bool) error {
 		return printFeaturesInASCIITable(features, true, wide)
 	case Column:
 		return printFeaturesInASCIITable(features, false, wide)
+	case YAML:
```

Comment:
can we add a test case in integration test?

Reply:
done